docs/sphinx/source/fsd/index.md

```diff
@@ -0,0 +1,64 @@
+
+### Properties of the BAG
+
+The Navigation Surface concept requires that in addition to estimation of depth, an estimate of the uncertainty associated with the depth must be computed and preserved.  In order to make the system suitable to support safety of navigation applications, there is a means to over-ride any automatically constructed depth estimates with "Hydrographer Privilege", (essentially, a means to specify directly the depth determined by a human observer as being the most significant in the area, irrespective of any statistical evidence to the contrary).
+
+There is also the requirement to provide data on the data (i.e., metadata, which describe all aspects of the data’s life, from methods of capture to processing methods, and from geospatial extents to responsible party).
+
+Finally, there must be a means to certify that the data in the file has been inspected by someone with appropriate experience and authority, that the data have been verified as suitable for some specific purpose, and that the file has not changed since this certification was made: in essence, a digital replacement for the Hydrographer’s signature.
+
+Means to incorporate all of these requirements in a portable, extensible, platform neutral, vendor neutral format are provided in the remainder of this Format Specification Document (FSD).
+
+### Status of the Project
+
+The ONSWG maintain both a [website](http://www.opennavsurf.org), which contains the current recommended release of the source code and supporting documentation, and a [Git repository](https://github.com/OpenNavigationSurface/BAG), which contains the source code of the BAG library.
+
+The BAG library is designed to be built from source code, including all component libraries that the BAG uses.  All component libraries are covered under open-source licenses, although not all the licenses are alike. Users should ensure that their use of the BAG library, and its dependencies, is compliant with the terms of the appropriate license before using the component libraries.  The ONS source code library itself is licensed under the [3-Clause BSD License](https://opensource.org/licenses/BSD-3-Clause) and you are free to modify, adapt and otherwise reuse the source code, including the construction of derivative products based on the source code, so long as you do so in accordance with the terms of the 3-Clause BSD license.  However, if a bug is found or there is a way to improve part of the library, it is requested that this be communicated to the ONSWG as described in the [BAG Format Extensions](FSD-Extensions.md) chapter.  Extensions to the library shall not be added without permission of the [BAG Architecture Review Board](FSD-BAGARB.md).  Applications for extensions are also covered in the [BAG Format Extensions](FSD-Extensions.md) chapter.
```

Comment:
"Extensions to the library shall not be added without permission of the [BAG Architecture Review Board](FSD-BAGARB.md)." is not aligned with the BSD-3-Clause. Do we mean "extensions to the format"? Even in that case, isn't the FSD under the same BSD-3-Clause?

Reply:
Well, since only ONS-approved people have write access to the repo that contains both the FSD and the code, I think this implicitly means permission is required from the ARB for such changes to be made to the official spec or reference implementation. @brian-r-calder @GlenRice-NOAA thoughts?